doc/guide/authentication.xml

```diff
@@ -99,4 +99,75 @@
 
     </section>
   </section>
+
+  <section id="twofa-auth">
+    <title>Enable two-factor authentication</title>
+
+    <para>As with any other <code>pam</code> based service that supports it,
```

Comment:
The official spelling is PAM. This could also link to http://www.linux-pam.org/Linux-PAM-html/Linux-PAM_SAG.html

Reply:
Got it, I will correct this.